py_expression_eval/tests.py

```diff
@@ -13,6 +13,7 @@
 
 import unittest
 
+# from py_expression_eval import Parser
```

Comment:
Remove this line

Reply:
I think I've made and pushed the change you requested where I deleted a commented out import line.